### PR TITLE
update refractr ingress-nginx helm to latest version

### PR DESCRIPTION
**Pull request by cmharlow**

### CODE REVIEW

File: terraform/main.tf [hunk 1]
```diff
@@ -47,6 +47,7 @@ module "itse-apps-prod-1" {
   external_secrets_settings   = local.external_secrets_settings
   node_groups                 = local.node_groups
   fluentd_papertrail_settings = local.fluentd_papertrail_settings
+  enable_logging              = true
```
Comment (floatingatoll) [inline terraform/main.tf:50]: Is this included intentionally? Seems entirely possible, but I figured I'd check.

**Review by floatingatoll — APPROVED** [review]
Seems the same as the stage commit from yesterday's work.

File: terraform/main.tf (hunk 1 above)
Comment (cmharlow) [reply terraform/main.tf:50]: yes

Comment (cmharlow) [inline terraform/main.tf:50]: bookmark for docs

**Pull request merged**